README.md

```diff
@@ -68,26 +68,46 @@ To run Knot.x you only need Java 8.
 
 To build it you also need Maven.
```

Comment:
Maybe it's time to add some Table of Contents to README file?
It may be easily generated with [npm md-toc](https://www.npmjs.com/package/md-toc).